styles.css

```diff
@@ -17,7 +17,6 @@ time, mark, audio, video {
 }
 
 body{
-background-color: white;
```

Comment:
What is the reason for removing this? what if some browser has a default background? Do you want to have a safe-guard here and have a default background color?

Reply:
I was messing around with styles. But speaking of browser default background, I would leave it as is. I think it very small issue which could happen very likely.